madanalysis/install/install_fastjetcontrib.py

```diff
@@ -85,7 +88,9 @@ def Unpack(self):
 
     def Configure(self):
         # Input
-        theCommands=['./configure','--fastjet-config='+self.bindir]
+        # TODO: figure out how to give `-std=c++11 -fPIC` together to CXXFLAGS
+        # using " or ' doesn't work on linux systems
+        theCommands = ['./configure', '--fastjet-config=' + self.bindir, 'CXXFLAGS=-fPIC']
```

Comment:
**TODO:** fjcontrib still needs to be compiled with `CXXFLAGS="-std=c++11 -fPIC"`; however couldn't figure out how to feed them together to the terminal. The configuration file doesn't accept `"` or `'` through the python interface, although both work when done by hand.

Reply:
Have you tried with ```\'```?

Reply:
yeap didn't work for me even tried the Unicode string :/